jobserver/api/jobs.py

```diff
@@ -119,6 +119,9 @@ def post(self, request, *args, **kwargs):
 
                     span.set_attribute("db_id", str(job.id))
                     span.set_attribute("job_identifier", job.identifier)
+                    span.set_attribute("backend", job_request.backend)
```

Comment:
This will be the related Backend object, is that expected?

Reply:
yeah, maybe I want `job_request.backend.slug`?

Reply:
(I guess I was hoping something along the line would convert this to a string for me)

Reply:
https://github.com/opensafely-core/job-server/pull/1717 perhaps?